cli/src/commands/help.rs

```diff
@@ -26,21 +30,94 @@ pub(crate) struct HelpArgs {
     pub(crate) command: Vec<String>,
 }
 
+struct Category {
+    description: &'static str,
+    content: &'static str,
+}
```

Comment:
nit: I would move this type after `cmd_help()` so `HelpArgs` is placed closer to the consumer function.